packages/utilities/src/math.ts

```diff
@@ -8,4 +10,38 @@ export function getDistanceBetweenPoints(point1: IPoint, point2: IPoint): number
   let distance = Math.sqrt(Math.pow(point1.x - point2.x, 2) + Math.pow(point1.y - point2.y, 2));
 
   return distance;
-}
\ No newline at end of file
+}
+
+export interface IFitContentToBoundsOptions {
```

Comment:
add jsdoc comments for the options and the method? do we need unit tests?

Reply:
I added unit tests and documentation.